gitops/argocd/tektoncd/tekton-config.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  profile: all
+  targetNamespace: openshift-pipelines
+  pruner:
+    resources:
+      - taskrun
+    keep: 100
+    schedule: "0 8 * * *"
```

Perhaps add a YAML comment here explaining the schedule. Cron format isn't exactly the easiest to read.

```suggestion
    # Run the TaskRun pruner every day at 8:00 AM
    schedule: "0 8 * * *"
```

---

I added a reference from wikipedia, I hope that would be okay.